src/cosmosdb-continuous-backup/azext_cosmosdb_pitr/_help.py

```diff
@@ -0,0 +1,151 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['cosmosdb create'] = """
+type: command
+short-summary: Creates a new Azure Cosmos DB database account.
+parameters:
+  - name: --locations
+    short-summary: Add a location to the Cosmos DB database account
+    long-summary: |
+        Usage:          --locations KEY=VALUE [KEY=VALUE ...]
+        Required Keys:  regionName, failoverPriority
+        Optional Key:   isZoneRedundant
```

Comment:
could you also add allowed values for these properties?

Reply:
There are three allowed keys: regionName, failoverPriority and isZoneRedundant.
regionName is just the name of the region. The allowed values are not constant. So its not possible to add them here. 
The allowed values of failoverPriority are already mentioned in the description. 
isZoneRedundant is just a boolen. 
Do you think we need any more explanation?